### PR TITLE
Make links clickable

### PR DESCRIPTION
**Pull request by siepra**

### CODE REVIEW

File: packages/desktop/src/renderer/components/widgets/channels/NestedMessageContent.tsx [hunk 1]
```diff
@@ -34,10 +43,19 @@ export interface NestedMessageContentProps {
...
+    return (
+      <a onClick={() => { openUrl(decoratedHref) }} className={classNames({ [classes.link]: true })} key={key}>
+        {decoratedText}
+      </a>
+    )
+  }
+
   return (
     <Grid item>
       {message.type === 2 ? ( // 2 stands for MessageType.Image (cypress tests incompatibility with enums)
```
Comment (Dinek007) [inline packages/desktop/src/renderer/components/widgets/channels/NestedMessageContent.tsx:61]: were is case for message.type 4? and for image size from my last changes here?

Comment (Dinek007) [reply packages/desktop/src/renderer/components/widgets/channels/NestedMessageContent.tsx:61]: never mind its compered with master

**Review by Dinek007 — APPROVED** [review]

File: packages/desktop/src/renderer/components/widgets/channels/NestedMessageContent.tsx (hunk 1 above)
Comment (siepra) [reply packages/desktop/src/renderer/components/widgets/channels/NestedMessageContent.tsx:61]: This PR points to master, not develop xD

Comment (siepra) [reply packages/desktop/src/renderer/components/widgets/channels/NestedMessageContent.tsx:61]: Develop is not yet merged

**Review by holmesworcester — APPROVED** [review]

**Pull request merged**